packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -18,7 +18,7 @@ import 'view.dart';
 /// Signature for determining whether the given data will be accepted by a [DragTarget].
 ///
 /// Used by [DragTarget.onWillAccept].
-typedef DragTargetWillAccept<T> = bool Function(T? data);
+typedef DragTargetWillAccept<T> = bool Function(T data);
```

Comment:
This would break customers by requiring them to change the function signature and then they would need to update their implementations to handle when data is null.